kasa/smart/smartdevice.py

```diff
@@ -36,6 +40,11 @@
 # same issue, homekit perhaps?
 WALL_SWITCH_PARENT_ONLY_MODULES = [DeviceModule, TimeModule, Firmware, CloudModule]
 
+AVAILABLE_BULB_EFFECTS = {
+    "L1": "Party",
+    "L2": "Relax",
+}
```

Comment:
I think we will probably want to get rid of the effects from the main interface, this was never really working for SMART devices anyway :-)

Reply:
I wonder if we might want to keep it in the main interface as the light platform in HA supports effects and I'm sure we'd be able to get this working,

Reply:
Okay, so, let's keep the interface for now, but remove the implementation part from `SmartDevice` as it's not really working. The future implementation should be done in a module (component `light_effect`).

Reply:
I wasn't completely sure which bits to pull out and which to keep so for now I've kept it as is but will do the `LightEffectModule` next.

Reply:
Okay, cool!